doc/org.eclipse.cdt.doc.user/src/asciidoc/core_build_system.adoc

```diff
@@ -0,0 +1,57 @@
+////
+Copyright (c) 2000-2024 QNX Software Systems and others
+This program and the accompanying materials
+are made available under the terms of the Eclipse Public License 2.0
+which accompanies this distribution, and is available at
+https://www.eclipse.org/legal/epl-2.0/
+
+SPDX-License-Identifier: EPL-2.0
+////
+
+
+// support image rendering and table of contents within GitHub
+ifdef::env-github[]
+:imagesdir: ../../images
+:toc:
+:toc-placement!:
+endif::[]
+
+// enable support for button, menu and keyboard macros
+:experimental:
+
+= Core Build System
+
+The Core Build System is the successor of the Managed Build System.
```

Comment:
Does everyone agree with this messaging? Personally, I view Core Build more as an alternative rather than a successor at present. But that could change over time.

Cc: @jonahgraham

Reply:
Since I totally missed the moment when MBS became a "legacy" and Core Build became a 
"favorite" here is the discussion to collect opinions https://github.com/eclipse-cdt/cdt/discussions/961

Reply:
I got this idea when reading old discussions with Dough Schaefer. You are right. We keep Managed Build. I will change it.

Reply:
CDT "deprecated" MBS many many years ago and @ewaterlander's messaging is in line with what was decided then - see this release review from 2017 as an example https://projects.eclipse.org/projects/tools.cdt/releases/9.3.0/review - quoted here:

> We have a few systems that are duplicated legacy and new. We have the new Core Build system that should be a long term replacement for the Managed Build system. Until we can do everything with the new one, we have both systems remaining.

Reply:
And the follow on release review quoted here https://projects.eclipse.org/projects/tools.cdt/releases/9.4.0/review:

> As Core Build matures, and with Managed Build largely unmaintained, we have started planning for Managed Build's deprecation. Integrators adding in new build integrations should look at Core Build.